Jenkinsfile

```diff
@@ -1,32 +1,34 @@
 GKERegion='us-central1-a'
 
```

Comment:
could move this var to environment vars? It would be easier to see all vars together)  Not critical

Reply:
Let's leave this for some next PR.